charts/fluent-bit-out-hsdp/Chart.yaml

```diff
@@ -1,17 +1,21 @@
-apiVersion: v1
+apiVersion: v2
 name: fluent-bit-out-hsdp
 description: Fast and lightweight log processor and forwarder or Linux, OSX and BSD family operating systems.
 keywords:
   - logging
   - fluent-bit
   - fluentd
   - hsdp
-version: 0.0.13
-appVersion: 1.9.9-5
+version: 0.0.14
+appVersion: 1.9.9
```

Comment:
I suggest our plugin uses its own semver release cadence.

Reply:
Yeah, I'll diverge on the next iteration, right now we just happen to be in sync for historic reasons :)